src/content/docs/en/guides/cms/kontent-ai.mdx

```diff
@@ -0,0 +1,514 @@
+---
+title: Kontent.ai & Astro
+description: Add content to your Astro project using Kontent.ai as a CMS
+type: cms
+service: Kontent.ai
+i18nReady: true
+---
+import FileTree from '~/components/FileTree.astro'
+import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
+
+
+
+[Kontent.ai](https://www.kontent.ai/) is the headless CMS that allows you to manage content in a structured and modular way, comes with AI capabilities, and delivers content to any platform you choose.
```

Comment:
Removed `delivers content to any platform you choose`, since that's a duplicate of `Headless CMS`
```suggestion

[Kontent.ai](https://www.kontent.ai/) is a headless CMS that allows you to manage content in a structured and modular way, supported by AI capabilities.
```